### PR TITLE
custom_script: Make the 'args' parameter optional

### PR DESCRIPTION
**Pull request by dsemenets**
Moulin didn't work if 'args' parameter absent. Make that paremeter optional

### CODE REVIEW

**Comment by lorc** [comment]
Please fix the spelling in the commit message and add test for all the cases.

**Comment by lorc** [comment]
There is a test framework enabled in Moulin to run automated unit tests:

https://github.com/xen-troops/moulin/tree/main/tests

You need to implement unit tests for `args` handling.

**Comment by dsemenets** [comment]
Added tests

File: tests/integration_tests/custom_script_builder/script.sh
Comment (lorc) [inline tests/integration_tests/custom_script_builder/script.sh:1]: Could you please move this script to the "resources" directory as well?

Comment (dsemenets) [reply tests/integration_tests/custom_script_builder/script.sh:1]: Done

**Pull request closed without merging**